lib/_stream_readable.js

```diff
@@ -153,6 +153,10 @@ function ReadableState(options, stream, isDuplex) {
   // Indicates whether the stream has finished destroying.
   this.closed = false;
 
+  // True if close has been emitted or would have been emitted
+  // depending on emitClose.
+  this.closeEmitted = false;
```

Comment:
Is this needed? There are no tests for this.

Reply:
removed